src/shared/icon/icon.css.ts

```diff
@@ -4,6 +4,7 @@ const styles = css`
   :host {
```

Comment:
Variabelen toevoegen?

--icon-color: var(--text-color-base)
--icon-surface-color: var(--surface-current);

Reply:
Alleen als je ze optioneel niet contextueel wil hebben. Dat was wel mijn uitgangspunt. Aan de ene kant omdat het icoon de --text-color-base van zijn context erft. En ook omdat --surface-current een contextuele variabele is. Een parent behoort dit te zetten. Die heeft een surface kleur.

Daarmee kan ik me alleen van de --icon-color iets voorstellen. Dit zou dan iets worden als

```
:host {
  --_icon-color: var(--icon-color, var(--text-color-base));
  
  color: var(--_icon-color);
  …
}
```